doc/JSON_INFO.md

```diff
@@ -216,10 +216,10 @@ The syntax listed here is still valid.
   [ "fur", 1 ]
 ],
 [
-  [ "plant_fibre", 20 ],
-  [ "sinew", 20 ],
-  [ "thread", 20 ],
-  [ "duct_tape", 20 ]
+  [ "plant_fibre", 20, false ], // Optional flag for recoverability, default is true.
```

Comment:
Could the flag be some kind of string like "NO_RECOVER", instead? False doesn't exactly tell me what I'm looking at here, unless I read the docs (which nobody ever does).